### ISSUE TITLE
Can you store images by the first few letters of the image?

### ISSUE THREAD

**Issue by lin0kin** [Sergei-Korneev/obsidian-local-images-plus#27]
like i got a image called `01234.jpg`, and this will store in `_resources/0/1/01234.jpg`. and the `00123.jpg` will store in '_resources/0/0/0123.jpg`

i have 14852 images in valut, need this feature. plz consider it.

**Comment by Sergei-Korneev** [comment]
Hi. Well, the main idea behind using md5 is to avoid duplicate attachments in the vault. When users are allowed to change attachment names at will, this idea will not make sense.

**Comment by lin0kin** [comment]
> Hi. Well, the main idea behind using md5 is to avoid duplicate attachments in the vault. When users are allowed to change attachment names at will, this idea will not make any sense.

yeah, you're right, i just want some features which i want. for example, i have png file called `a2ba21002a21bca05b8f627b824021fd.png`.

1. Plz consider to allow user to define the date pattern, like set the media folder to `_resources/${date:YYYYMM}`, then attachment's full path will be  `_resources/202306/a2ba21002a21bca05b8f627b824021fd.png`. I want year+month `YYYYmm` to be subfolder's name, not `dd.mm.yyyy`. Saving the day is meaningless to me.

2. Not asking to giveup the md5, i really dont want the string `_MD5` in attachment name's ending, like: `a2ba21002a21bca05b8f627b824021fd.png` will be fine, `a2ba21002a21bca05b8f627b824021fd_MD5.png` is not ok. Dont understand the meaning of the string `_md5`
